ui-particles-angular/projects/ui-policy-studio-angular/src/lib/components/flow-proxy-form-dialog/gio-ps-flow-proxy-form-dialog.component.ts

```diff
@@ -55,7 +54,7 @@ export class GioPolicyStudioFlowProxyFormDialogComponent {
       name: new FormControl(flowDialogData?.flow?.name ?? ''),
       pathOperator: new FormControl(httpSelector?.pathOperator ?? 'EQUALS'),
       path: new FormControl(httpSelector?.path ?? ''),
-      methods: new FormControl(httpSelector?.methods ?? ['ALL']),
+      methods: new FormControl(!httpSelector?.methods || httpSelector?.methods.length === 0 ? ['ALL'] : httpSelector?.methods),
```

Comment:
hum 🤔 
if `httpSelector?.methods` = [] this return [] no ? and not ['ALL']

Maybe we can add the opposite of sanitizeMethods like sanitizeFormControlMethods ?

Reply:
If we want 'ALL' to be selected in the tag input when loading the form, we need to add this. But I agree, I will add a method to compute this

Reply:
yes, because if we save and then reopen the dialog, the value will be ALL, so better to have it by default